### ISSUE TITLE
Difference with nuitka

### ISSUE THREAD

**Issue by cyw3** [marcelotduarte/cx_Freeze#1751]
Hi,
I want to know cx_Freeze's difference with [Nuitka](https://github.com/Nuitka/Nuitka).

Thanks!

**Comment by mouadessalim** [comment]
cx_Freeze and Nuitka are both tools for creating standalone executables from Python programs.

cx_Freeze is a set of scripts and modules for freezing Python scripts into executables in a cross-platform way, using many of the techniques found in Thomas Heller's py2exe, Gordon McMillan's Installer and the Freeze utility that ships with Python itself. It supports creating executables for Windows, Linux, and Mac OS X.

Nuitka is a python compiler which converts python code to C/C++ source code which can then be compiled to standalone executables. Nuitka aims to fully support the Python language, including all the standard libraries and even some of the most popular third-party libraries, and also provides additional performance improvements over the original python code. It also supports creating executables for Windows, Linux, and MacOS.

In summary, cx_Freeze is a simpler tool that uses existing python libraries to create executables, while Nuitka is a more advanced tool that compiles python code to C/C++ to create standalone executable.

**Comment by HydraDragonAntivirus** [comment]
That should be move to discussion.
